fsutil_test.go

```diff
@@ -106,6 +137,9 @@ func TestIsEmpty(t *testing.T) {
 			t.Errorf("Expected %t, Result was: %t", test.expected, result)
 		}
 	}
+
+	// Remove temp dir
+	os.RemoveAll(dir)
```

Comment:
It's better practice to use defer incase something else returns out of the function before the `RemoveAll` is called.